docs/image_loaders.md

```diff
@@ -49,7 +49,7 @@ However, if you'd prefer a singleton the `io.coil-kt:coil` artifact provides a s
 For instance, you could inject a fake `ImageLoader` implementation which always returns the same `Drawable` synchronously:
 
 ```kotlin
-val fakeImageLoader = object : ImageLoader {
+fun createFakeImageLoader(context: Context) = object : ImageLoader {
```

Comment:
Let's stick with `val` here to keep the callsite simple.

Reply:
Changed to property extension.

Reply:
I'd prefer to leave it as is since the code is meant to be used as an example to then call `Coil.setImageLoader` or inject the `fakeImageLoader` into your dependency graph.

Reply:
Ah I see - for the context for `ImageLoader.Builder` we can just have `ImageLoader.Builder(context)` and assume anyone copying the code can provide a context (this example code will probably be copied into a `TestApplication` class). That way we can keep `val fakeImageLoader`.

Reply:
Done 😄